test/fixtures.py

```diff
@@ -43,6 +41,9 @@
 INIT_DB_PATH = 0
 
 
+__all__ = ["init_db_path", "database_uri", "sqlitedb_file", "appconfig", "initialized_db", "app"]
```

Comment:
This package is typically used as `from test.fixtures import *`, without `__all__` it also provides `os`, `pytest`, `Flask`, `model`, etc.